sdk/template/template/karma.conf.js

```diff
@@ -20,33 +29,36 @@ module.exports = function(config) {
       "karma-ie-launcher",
       "karma-env-preprocessor",
       "karma-coverage",
-      "karma-sourcemap-loader",
-      "karma-junit-reporter"
+      "karma-remap-istanbul",
```

Comment:
@xirzec exterminated this dependency from our repo (see https://github.com/Azure/azure-sdk-for-js/pull/13309).

Reply:
Yes plz remove here too!

Reply:
@xirzec I got rid of it, but I don't think it was removed entirely from the pnpm-lock. Not sure if there's more for me to do on this.